feature/loan-account/src/commonMain/kotlin/org/mifos/mobile/feature/loanaccount/viewmodel/LoanAccountViewmodel.kt

```diff
@@ -7,6 +7,8 @@
  *
  * See https://github.com/openMF/mobile-mobile/blob/master/LICENSE.md
  */
+@file:Suppress("ktlint:standard:property-naming")
```

Comment:
Hi @Nagarjuna0033 ,

I see that you used `@file:Suppress("ktlint:standard:property-naming")` at the top of the file. Just a quick note when this annotation is placed at the file level, it suppresses all property naming violations in the entire file.

As we discussed before, when I ran it with `@Suppress("PropertyName")`on a specific property, I didn’t receive any Spotless errors. This approach could be a better choice since it only suppresses the rule for that particular property, and in the future, if someone adds a property that doesn’t follow naming conventions, it will still trigger a warning.

If `@Suppress("PropertyName")` doesn’t work for you, which worked for me, then you can try `@Suppress("ktlint:standard:property-naming")` directly on the specific property